tracer/test/Datadog.Trace.Tests/Configuration/TracerSettingsServerlessTests.cs

```diff
@@ -57,6 +48,40 @@ public void HttpClientExcludedUrlSubstrings(string value, bool isRunningInAppSer
         }
     }
 
+    // These tests rely on Lambda.Create() which uses environment variables
+    // See TracerSettingsTests for tests which don't rely on any environment variables
```

Comment:
Is this a lie? 🤔 Or are they used _internally_? Because if the latter, eww, that's gross, but a problem for another time I guess 😅